### PR TITLE
Chinese localization

### PR DESCRIPTION
**Pull request by cashlifei**
Add or improve Chinese translations

### CODE REVIEW

**Comment by LonwoLonwo** [comment]
Hello @cashlifei 

Thanks for your contribution!
We will review your PR and let you know if some changes are needed.

**Comment by LonwoLonwo** [comment]
Hello @cashlifei 

Some of your localized files are already mapped. 
Like webapp/packages/core-administration/src/locales/zh.ts
https://github.com/dbeaver/cloudbeaver/blob/devel/webapp/packages/core-administration/src/AdministrationLocaleService.ts#L30

But for all new zh files, you must add mapping to the corresponding folder, LocalServise.ts files.

https://github.com/dbeaver/cloudbeaver/blob/devel/webapp/packages/core-view/src/LocaleService.ts for /core-view as example. With it, your localization will be visible.

**Comment by LonwoLonwo** [comment]
I just wanted to remind you about the file mapping. The instruction you can find here: https://github.com/dbeaver/cloudbeaver/wiki/CloudBeaver-localization

Also, you are having conflicts now in your PR. Please fix it.

**Comment by cashlifei** [comment]
> I just wanted to remind you about the file mapping. The instruction you can find here: https://github.com/dbeaver/cloudbeaver/wiki/CloudBeaver-localization
> 
> Also, you are having conflicts now in your PR. Please fix it.

Already fix the conflicts and add mapping for new zh files. Thanks for your remind

File: webapp/packages/core-localization/src/locales/zh.ts [hunk 1]
```diff
@@ -101,36 +101,35 @@ export default [
...
+  ['ui_upload', '上传'],
+  ['ui_import', '导入'],
+  ['ui_view', '查看'],
+  ['ui_show_more', '查看更多'],
+  ['ui_limit', '限制'],
+  ['ui_file_size', '文件大小'],
+  ['ui_processing_synchronization', '同步...'],
+  ['ui_no_file_chosen', '未选择文件'],
+  ['ui_file_size_exceeds', '文件大小超限。文件大小: {arg:size}，最大文件大小: {arg:maxSize}'],
+  ['ui_upload_file', '上传文件'],
```
Comment (LonwoLonwo) [inline webapp/packages/core-localization/src/locales/zh.ts:122]: Just checking. You translated both (file and files) as "Upload files". 
Could you please translate "Upload file" with file word in singular?

Comment (cashlifei) [reply webapp/packages/core-localization/src/locales/zh.ts:122]: In Chinese, nouns do not distinguish between singular and plural

Comment (cashlifei) [reply webapp/packages/core-localization/src/locales/zh.ts:122]: For example, we say "one apple" and "two apples." In Chinese, there is no plural form for nouns. Instead, context is used to indicate whether a noun is singular or plural.

Comment (LonwoLonwo) [reply webapp/packages/core-localization/src/locales/zh.ts:122]: Oh, wow, thanks for the clarification.

**Review by LonwoLonwo — APPROVED** [review]

**Comment by LonwoLonwo** [comment]
Now, we are waiting for the second approval and approval from our QA team.

**Review by EvgeniaBzzz — APPROVED** [review]

**Review by alexander-skoblikov — APPROVED** [review]

**Comment by LonwoLonwo** [comment]
Thanks again for your contribution!
Your changes will be available in the next release 24.2.1

**Comment by LonwoLonwo** [comment]
Hello @cashlifei! Thank you for being with CloudBeaver! 
We appreciate your support and want to send you a compliment. Please write to an email [anna.pozdeeva@dbeaver.com](mailto:anna.pozdeeva@dbeaver.com) and provide your nickname in GitHub.

**Pull request merged**